pkgs/applications/virtualization/dumb-init/default.nix

```diff
@@ -11,7 +11,11 @@ stdenv.mkDerivation rec {
     sha256 = "sha256-aRh0xfmp+ToXIYjYaducTpZUHndZ5HlFZpFhzJ3yKgs=";
   };
 
-  buildInputs = lib.optionals (!stdenv.hostPlatform.isMusl) [ glibc.static ];
+  postPatch = lib.optionalString (!stdenv.hostPlatform.isStatic) ''
+    substituteInPlace Makefile --replace "-static" ""
+  '';
+
+  buildInputs = lib.optionals (!stdenv.hostPlatform.isMusl) (if stdenv.hostPlatform.isStatic then [ glibc.static ] else [ glibc ]);
```

Comment:
I think the `glibc` input for glibc&&!static is not needed?
If so, the line could be simplified to: `buildInputs = lib.optional (stdenv.hostPlatform.isGnu && stdenv.hostPlatform.isStatic) glibc.static;`

Reply:
Done.